src/app/pages/LoginPage/index.tsx

```diff
@@ -31,6 +31,6 @@ const styles = {
     flex
     justify-center
     w-full
-    bg-[rgba(255,255,255,0.6)] dark:bg-[rgba(91,112,131,0.4)]
+    bg-[rgba(0,0,0,0.4)] dark:bg-[rgba(91,112,131,0.4)]
```

Comment:
Seems strange that you changed from white to dark color

Reply:
Yes, because I changed the main background. It works correctly.